Makefile

```diff
@@ -15,13 +15,15 @@ OBJS =	alloc.o c_ksh.o c_sh.o c_test.o c_ulimit.o edit.o emacs.o \
 	var.o version.o vi.o setmode.o signame.o strlcat.o strlcpy.o \
 	reallocarray.o
 
+# set OS platform (HACKITY HACK for gmake/bsd make compat)
+OS != echo `uname -o` | tr 'A-Z' 'a-z' | sed -E 's/.*(bsd|linux)/\1/'
```

Comment:
OpenBSD doesn't have `uname -o` -- makes me wonder if others also don't have it?